packages/ketcher-react/src/script/editor/utils/selectItems/continueSelecting.ts

```diff
@@ -0,0 +1,121 @@
+import utils from '../../shared/utils'
+import {
+  fromArrowResizing,
+  fromMultipleMove,
+  fromSimpleObjectResizing,
+  getHoverToFuse,
+  getItemsToFuse
+} from 'ketcher-core'
+import { selMerge } from '../../tool/select'
+
+export function continueSelecting(event, self, lassoHelper) {
```

Comment:
code in onMouseMove in select.ts moved in continueSelecting function with little or no updates